packages/dx-grid-core/src/plugins/table-column-resizing/reducers.js

```diff
@@ -15,13 +15,13 @@ export const changeTableColumnWidth = (state, { columnName, shift }) => {
   };
 };
 
-export const changeDraftTableColumnWidths = (state, { shifts }) => {
+export const changeDraftTableColumnWidth = (state, { columnName, shift }) => {
   const { columnWidths } = state;
-  const updatedColumn = columnWidths.find(elem => !!shifts[elem.columnName]);
-  if (!updatedColumn) {
+  const updatedColumn = columnWidths.find(elem => elem.columnName === columnName);
+  if (!shift) {
```

Comment:
Why do we reset all shifts and not only for a given column?

Reply:
Because `draftColumnWidths have only one object`.
`draftColumnWidths: [{ columnName: updatedColumn.columnName, width: size }]`
And maybe should be renamed to `draftColumnWidth`?